corehq/motech/requests.py

```diff
@@ -266,3 +270,25 @@ def simple_post(domain, url, data, *, headers, auth_manager, verify,
         message = f'HTTP status code {response.status_code}: {response.text}'
         requests.notify_error(message)
     return response
+
+
+def sanitize_user_input_url_for_repeaters(url, domain, src):
+    try:
+        sanitize_user_input_url(url)
+    except (CannotResolveHost, InvalidURL):
```

Comment:
Under what conditions is `InvalidURL` raised? That seems like something that should produce an error rather than being ignored.